composer.json

```
@@ -6,7 +6,7 @@
         "MIT"
     ],
     "require": {
-        "neos/flow": "^4.0 || ^5.0",
+        "neos/flow": "^5.0 || ^6.0",
```

Given the above comment this should be changed to
```suggestion
        "neos/flow": "^6.0",
```